create-litr.Rmd

```diff
@@ -426,6 +426,24 @@ setup <- function(package_dir) {
     chunk_start <- "^(\n```+[a-zA-Z0-9_]+\n)"
     idx_block <- stringr::str_which(x, chunk_start)
     original_code <- knitr::knit_code$get()
+    # We first get indices of skipped chunks in original_code list
+    skipped_chunks <- which(sapply(original_code, function(x){
+      echo_attr <- attr(x, "chunk_opts")$echo
+      include_attr <- attr(x, "chunk_opts")$include
+      if(!is.null(echo_attr) | !is.null(include_attr)){
+        if(!is.null(echo_attr)){
+          return(!echo_attr)
+        } 
+        if(!is.null(include_attr)){
+          return(!include_attr)
+        }
+      } else{
+        return(FALSE)
+      }
+    }))
+    # Next we remove the indices of skipped chunks and sort for good measure
+    original_code_idx_fixed <- sort(setdiff(seq(length(original_code)), skipped_chunks))
```

Comment:
Why is `sort` needed... wouldn't `seq` always produce something in order?

Reply:
Good point, fixed.